.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Feature request
+about: Request a new feature to the plugin
+title: ''
+labels: feature
```

Comment:
Or better `externally requested`? 🤔 I guess sometime we will be those who request

Reply:
+1, why not, we can removed the label then

Reply:
yup, I think `externally requested` looks more premium Xd